include_core/omrport.h

```diff
@@ -748,6 +748,7 @@ typedef struct J9MemoryInfo {
 	 * When not in a cgroup, this will be identical to 'buffered' field above.
 	 */
 	uint64_t hostBuffered;
+	uint64_t swappiness; /* Indicates kernel aggressivenss (0-100) in swapping memory pages */
```

Comment:
Are `J9MemoryInfo.swappiness` and `OMRCgroupMemoryInfo.swappiness` different? If so, the comments should reflect the differences.

Reply:
The struct `J9MemoryInfo` captures memory statistics for the host whereas the same statistics for the `cgroup` limits are captured in the struct `OMRCgroupMemoryInfo`. I have added extra comments around both places for clarity.